deployments/loader/values.yaml

```diff
@@ -1,80 +1,242 @@
   worker:
+    # Configuration of the Secret used for worker authentication when
+    # communication with the control plane.
     registrationKeySecret:
-      name:
-      key:
+      # The secret name. `default-cluster-registration-key` is available
+      # when the control-plane and worker-plane are in the same cluster.
+      # This Secret is generated by cluster-manager-server as default.
+      # For more information, see [Install across Multiple Clusters](https://llmariner.ai/docs/setup/install/multi_cluster_production/).
+      name: default-cluster-registration-key
+      # The key name with a registration key set.
+      key: key
+
     tls:
+      # The flag to enable TLS access to the control-plane.
       enable: false
-    # If specified, use this as the address for accessing the control-plane services.
+
+    # If specified, use this address for accessing the control-plane.
+    # This is necessary when installing LLMariner in a multi-cluster mode.
+    # For more information, see [Install across Multiple Clusters](https://llmariner.ai/docs/setup/install/multi_cluster_production/).
     controlPlaneAddr: ""
 
-baseModels:
-models:
+# +docs:section=Loader
 
+# The interval time to load models.
 modelLoadInterval: 1m
+# Specify whether to load models once at startup time.
 runOnce: false
 
-huggingFaceSecret:
-  name:
-  apiKeyKey:
+# The following default values work if model-manager-server runs in the same namespace.
+modelManagerServerWorkerServiceAddr: model-manager-server-worker-service-grpc:8082
+
+# The list of base models to load into LLMariner.
+# For more information, see [Supported Open Models](https://llmariner.ai/docs/features/models/).
+#
+# For example:
+# baseModels:
+# - google/gemma-2b-it-q4_0
+# - meta-llama/Meta-Llama-3.1-8B-Instruct-q4_0
+#
+# +docs:property
+# baseModels: []
 
+# The list of fine-tuned or quantized models to load into Llmariner.
+#
+# For example:
+# models:
+# - model: my-model-name
+#   baseMode: google/gemma-2b-it-q4_0
+#   adapterType: "" # or lora, qlora
+#   QuantizationType: "" # or gguf, awq
+#
+# +docs:property
+models: []
+
+# The configration for downloading models.
 downloader:
+  # The kind name indicating where the downloader gets models from.
+  # +docs:enum=s3,huggingFace
   kind: s3
+
+  # TODO(kenji): Switch to llmariner-models
+
+  # The configuration used when get models and base-models from s3.
   s3:
+    # The s3 endpoint URL.
     endpointUrl: https://s3.us-west-2.amazonaws.com
+    # The region name where the models are stored.
     region: us-west-2
-    # TODO(kenji): Switch to llmariner-models
+    # The bucket name where the models are stored.
     bucket: llm-operator-models
+    # The path prefix of the model.
     pathPrefix: v1/base-models
+
+  # The configuration used when get models and base-models from huggingface..
   huggingFace:
-    cacheDir:
+    # The path to the cash directory for the huggingface hub.
+    cacheDir: "/home/.cache/huggingface/hub"
```

Comment:
I based this on the code, but I don't feel very confident about it. Could your review focus on this part? Are there any references for the `.models` configuration and model naming(? or ID) format?

Reply:
Thanks! The value of `cacheDir` looks good. 

We don't have the concrete documentation on the model ID format yet. This mostly follows Hugging Faces but not exactly when models are quantized. That's something we need to work on separately..

Reply:
Ah, sorry, I mentioned lines from 74 to 119. 

One question: since we need to specify either `adapterType` or `quantizationType` for models, in this example where the base model employs quantization, is it correct to set `quantizationType` to `gguf`?
```yaml
models:
- model: my-model-name
  baseMode: google/gemma-2b-it-q4_0
  adapterType: ""
  quantizationType: "gguf"
```

Reply:
Ah sorry.  :(

This is currently not fully sorted out yet, but 

- `baseModel` shouldn't be a quantized one. 
- `quantizationType` should be set to the quantization of the (q)Lora adaptor, but it is not currently used anywhere? Actually I'm not very certain on this. @guangrui-cloudnatix ?

```yaml
# models:
# - model: my-model-name
#   baseModel: google/gemma-2b-it
#   adapterType: "" # or lora, qlora
#   quantizationType: "" # or gguf, awq
```

Reply:
my understanding is: 
`model`: the name of the model, which should match with the model name in the inference request
`baseModel`: it depends on the type of the `model`:

-  For `quantized` model, e.g. 

```models:
- model: google/gemma-2b-it-q4_0
  baseMode: google/gemma-2b-it
  adapterType: ""
  quantizationType: "gguf"```

- For `adapter` model, the baseModel can be a quantized model or plain based model. 

The reasoning to have such fields are: in case of vllm, which supports loading multiple lora adapters if they have the same base model (with the same quantization), we can use `baseModel` to check if loras can be loaded in the same runtime. we can use `quantization` type to check if a model is supported by the engine, we haven't fully check this yet.

Reply:
thx!